src/rt.erl

```diff
@@ -1213,9 +1209,12 @@ product(Node) ->
        true -> unknown
     end.
 
-try_nodes_ready([Node1 | _Nodes], 0, _SleepMs) ->
-    lager:info("Nodes not ready after initial plan/commit, retrying"),
-    plan_and_commit(Node1);
```

Comment:
Yeah, definitely agree. I figured even if no other benefit ultimately comes from this PR, I at least wanted to kill off that particular bit of weirdness :)